certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/Certificate.java

```diff
@@ -20,7 +21,8 @@ public class Certificate {
   private final CertificateModel certificateModel;
   private final LocalDateTime created;
   private CertificateMetaData certificateMetaData;
-  private List<ElementData> elementData;
+  @Builder.Default
+  private List<ElementData> elementData = new ArrayList<>();
```

Comment:
Thinking that the default value should be an immutable list. So there is no chance that "someone" adds things to it. It should only be updated with updateData(...) and when the Certificate is being built.

Reply:
If I understand correct, we want to remove the Builder.Default annotation.

In some scenarios, like when we create or get a certificate, the element data list might not have been initialized and is null. If we dont want to use the Builder.Default-annotation, we need to introduce a null check, something like this:

CertificateConverter:
```java
      .data(
            certificateDataConverter.convert(
                certificate.certificateModel(),
                certificate.elementData() != null ? certificate.elementData() : Collections.emptyList()
            )
        )
```

Is that a more appropriate way to handle the chance of ElementData beeing null?

Alternatively, we could introduce a get-function in Certificate to get ElementData:

```java
  public List<ElementData> getElementData() {
    if (this.elementData == null) {
      return Collections.emptyList();
    }
    return this.elementData;
  }
```

Reply:
No, I didn't mean to remove the default-annotation. Instead the default value should be an immutable list. 

If someone updates the elementData, it would only be through the updateData(...) and that replaced the current list.

Reply:
Ah I see, great catch!